### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by creepinson** [igoradamenko/esbuild-plugin-alias#1]
This package will not work with typescript due to there being no .d.ts files in the project. 

I have created a .d.ts file myself so it works locally:

```typescript
declare module "esbuild-plugin-alias" {
    import type { Plugin } from "esbuild";
    import alias from "esbuild-plugin-alias";

    const aliasTyped: (opts: Record<string, string>) => Plugin = alias;

    export default aliasTyped;
}
```
I will make a PR for this later today.

**Comment by igoradamenko** [comment]
> I will make a PR for this later today.

It would be nice, thanks.

**Comment by iron-cherep** [comment]
@creepinson @igoradamenko is this issue will be resolved soon by any chance? 🙂

**Comment by igoradamenko** [comment]
Landed in esbuild-plugin-alias@0.2.0.
